mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala

```diff
@@ -25,12 +25,12 @@ case class CompletionPos(
 
   def toEditRange: l.Range =
     def toPos(offset: Int): l.Position =
-      new l.Position(
-        cursorPos.source.offsetToLine(offset),
-        cursorPos.source.column(offset)
-      )
+      val line = cursorPos.source.offsetToLine(offset)
+      val column = offset - line
+      new l.Position(line, column)
```

Comment:
Could you explain what was wrong? I cannot comprehend this by myself :D

Reply:
This looks a bit weird. Why would be subtracting line number from file offset :thinking:

Reply:
oops. updated

@kpodsiad  the actual issue is that dotty's `SourceFile.column` method treats tabs incorrectly. If line starts with tabs they just don't count as symbols that results to having a wrong editRange in completionItems.

I will look how to fix that in dotty but we anyway need to fix in Metals for existing scala3 versions.